portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,20 @@ function addRandomFunFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+var i = 1;
```

Comment:
currentlyDisplayedProjectIndex?

I know the name might take up more space but when you come back to work on this later the more descriptive name will help remind you what this variable was for.

Reply:
Is it ok to use currentProjIndex since by HTML element is called currProj?

Reply:
sure!